server/db/schemas/statuses.sql

```diff
@@ -2,7 +2,7 @@ CREATE TABLE statuses (
     status_id SERIAL,
     patient_id INT NOT NULL,
     status JSON NOT NULL,
-    created_on TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT current_timestamp,
+    created_on TIMESTAMP WITH TIME ZONE NOT NULL,
```

Comment:
We should define the created on dates in the service layer so we have more control over them

Reply:
I was thinking the current time to be the fallback value if we don't define a time up to you if you want to remove it tho

Reply:
We could do that but its best to keep a single source of truth for date generation. basically we just make sure its consistent everywhere so no subtle bugs arrive